cfe_internal/enterprise/CFE_hub_specific.cf

```diff
@@ -765,27 +776,47 @@ bundle agent cfe_internal_enterprise_policy_analyzer
 
       "policy_analyzer_enabled" expression => fileexists( $(analyzer_flagfile) );
 
-    files:
+  files:
 
-    policy_analyzer_enabled::
+    enterprise_edition.policy_server::
 
-      "$(analyzer_dir)/."
+      "$(cfe_internal_hub_vars.docroot)/analyzer/."
         create => "true",
-        perms => mog( "500", $(def.cf_apache_user), $(def.cf_apache_user) );
+        handle => "cfe_internal_setup_knowledge_docroot_analyzer_dir",
+        perms => mog( "0470", "root", $(def.cf_apache_group) ),
+        comment => concat( "This directory holds the policy analyzer flag file",
+                           "that is written by Mission Portal when someone",
+                           "clicks to enable the feature. Thus, it needs to be",
+                           "readable, writeable and executable web",
+                           "server.");
```

Comment:
Missing "***for the** web server*"? Also, missing spaces at ends or beginnings of lines.

Reply:
ACK